src/Core/Misc/ExtensionMethods.cs

```diff
@@ -420,11 +420,7 @@ public static ScanStatus GetAggregatedScanStatus(this IEnumerable<ScanStatus> ts
                 }
                 else if (HasTestResults(tss, ScanStatus.NoResult))
                 {
-                    // it means that there was no rule executed or 
-                    if (System.Diagnostics.Debugger.IsAttached)
-                    {
-                         System.Diagnostics.Debugger.Break();
-                    }
+                    return ScanStatus.NoResult;
```

Comment:
@RobGallo, did we decide to return ScanStatus.NoResult (changing the behavior) or just to continue to let this return ScanStaus.Pass (keeping the current behavior) ?

Reply:
I would continue the prior behavior.

Reply:
Done